contracts/connectors/loantoken/LoanTokenLogicStandard.sol

```diff
@@ -756,9 +760,9 @@ contract LoanTokenLogicStandard is AdvancedToken, StorageExtension, Flags {
                     loanDataBytes,
                     (uint128, bytes[])
                 );
-                (address owner, address spender, uint value, uint deadline, uint8 v, bytes32 r, bytes32 s) = abi.decode(payload[2], (address, address, uint, uint, uint8, bytes32, bytes32));
+                (address owner, address spender, uint value, uint deadline, uint8 v, bytes32 r, bytes32 s) = abi.decode(payload[0], (address, address, uint, uint, uint8, bytes32, bytes32));
```

Comment:
0 index slot is reserved for dex selector. It needs to be 2 index slot as that is the next unused one.

Reply:
fixed